pkg/scheduler/core/generic_scheduler.go

```diff
@@ -218,6 +210,13 @@ func (g *genericScheduler) Schedule(ctx context.Context, state *framework.CycleS
 			FilteredNodesStatuses: filteredNodesStatuses,
 		}
 	}
+	trace.Step("Computing predicates done")
+
+	// Run "postfilter" plugins.
+	postfilterStatus := g.framework.RunPostFilterPlugins(ctx, state, pod, filteredNodes, filteredNodesStatuses)
```

Comment:
Can you help to rename:

- postfilter -> postFilter
- prefilter -> preFilter

across this file?

Reply:
This is a tricky one, I think it is still useful to run PostFilter before exit, the purpose of PostFilter is informational, perhaps to export logs/metrics, which one may want to do in all cases.

However, PostFilter is supposed to perform "pre-score" computations, and PostFilter plugins that does that should return directly if "filteredNodes" is empty. Otherwise, we will have to add a PreScore extension point.

I suggest we keep things as is, and just better document the expected PostFilter behavior when used as pre-score: when used as a pre-score, the plugin should skip doing any computations in casein case filteredNodes is empty since the Score phase will not be executed.

Reply:
Thanks @ahg-g I understand what you mean, that makes sense.
I agree that it's necessary to add a `PreScore` extension point.
IMO, `PostFilter` should only handle filter, not score. 
Otherwise, maybe we should call it `PostFilterAndPreScore` :man_shrugging:

Reply:
> 
> 
> Can you help to rename:
> 
>     * postfilter -> postFilter
> 
>     * prefilter -> preFilter
> 
> 
> across this file?

Thanks， I will update.